rules/privatization/src/Rector/Class_/ChangeReadOnlyVariableWithDefaultValueToConstantRector.php

```diff
@@ -173,8 +173,10 @@ private function filterOutUniqueNames(array $assigns): array
         }
```

Comment:
This method can be extracted with return type to reach the same goal.

Reply:
implemented.